guides/v2.2/cloud/configure/setup-cron-jobs.md

```diff
@@ -40,24 +25,54 @@ crons:
 ```
 
 {:.bs-callout .bs-callout-info}
-We use only this one cron for cloud due to the read-only nature of the environments. This is different from {{site.data.var.ee}} which has three default cron jobs.
+We use only this one cron for cloud projects due to the read-only nature of the environments. This configuration is different from {{site.data.var.ee}} which has three default cron jobs. See [Configure and run crons]({{ page.baseurl }}/config-guide/cli/config-cli-subcommands-cron.html) in the {{ site.data.var.ee }} documentation.
+
+## Verify that crons are working
+
+You can use the `crontab` command to view the cron processes configured in the `magento.app.yml` file.
```

Comment:
It would be better to state the step action without calling a command or a pronoun.

```suggestion
View the cron processes configured in the `magento.app.yml` file.
```

Reply:
Actually, I misread. I thought I was looking at a step. Yeah, this is almost repetitive, in a way.